android/app/src/main/kotlin/tech/himanshusharma/relic_bazaar/Application.kt

```diff
@@ -0,0 +1,18 @@
+package tech.himanshusharma.relicbazaar
```

Comment:
The Flutter version is already greater than 1.12, so no need to integrate FCM in such a way.